process-deleted-files.sh

```diff
@@ -46,11 +45,16 @@ function delete_csvw_outputs {
 
         git stash
 
+        local stash_content=$(git stash list)
```

Comment:
Actually, I think you should run this *after* you've checked out into `$GITHUB_REF_NAME`